apps/web/src/composables/auth.ts

```diff
@@ -33,7 +34,47 @@ export default function useAuth() {
         }
     }
 
+    function addAccount(provider: ProviderString, address: string) {
+        address = address.toLowerCase()
+        const localStorage = window.localStorage
+        const accounts = JSON.parse(localStorage.getItem('accounts') as string) || {}
+
+        // Check if the address already exists on a different added provider (remove if so)
+        for (const existingProvider in accounts) {
+            if (accounts[existingProvider].includes(address) && existingProvider !== provider) {
+                accounts[existingProvider] = accounts[existingProvider].filter((existingAddress: string) => existingAddress !== address)
+            }
+        }
+
+        // Add the new provider with address OR add the address to the existing added provider
+        if (!accounts[provider] && address) {
+            accounts[provider] = [address]
+        } else if (address) {
+            if (!accounts[provider].includes(address)) {
+                accounts[provider].push(address)
+            }
+        }
+
+        localStorage.setItem('accounts', JSON.stringify(accounts))
```

Comment:
Can you abstract the local storage methods to their own methods in a users composable? (updateUser(input))

You could create the updateUser input here in auth.

Reply:
@ccali11 can you address this?

Reply:
Addressed.